### ISSUE TITLE
No ores in Just Another Mining Dimension

### ISSUE THREAD

**Issue by sgonzales3478** [FTBTeam/FTB-Modpack-Issues#5291]
### Modpack

FTB Presents Direwolf20 1.20

### Modpack version

1.12.1

### Pack has been modified

- [ ] Yes

### Describe the bug

I crafted the block for Just Another Mining Dimension, but when I went into the mining world and started to mine there was no ores whatsoever! The world has only grass, dirt, stone. I have tried an older version of JAMD, tried deleting the dimension data to reset it, but neither worked and the dimension still generates with no ores whatsoever!

### Steps to reproduce

1. craft the JAMD block
2. Place the block and click on it to join the mining world
3. Start mining and find no ores in the mining dimension

### Expected behaviour

I expected to join the mining world and be able to start mining for ores, but the mining dimension had no ores.

**Comment by Gaz492** [comment]
Is this on a fresh install of the modpack or one you have had for a while and updated from an older version of the modpack?
I have so far not been able to reproduce the issue you are having.

If you enable allow cheats/op your self if on a server could you run the command `/tk clear`, this will remove all the stone/dirt/grass from ground level to bedrock. Does any ores show up in that cleared area for you?

**Comment by Gaz492** [comment]
Ok, for testing so we get some logs, would you be able to create a new world, spawn in the JAMD block go to the mining world then run the command again to see if any ores have spawned, then upload the latest.log and debug.log for us?
https://docs.feed-the-beast.com/docs/app/Instances/instance-logs

This may give us some info about whats going wrong for you.

**Comment by sgonzales3478** [comment]
Sorry for the delay......working at the same time that I'm trying to get the information for you.

I created the new world and performed what you mentioned and still no ores. Where do I find the two logs you were asking for? Not sure where to locate them.

**Comment by sgonzales3478** [comment]
My bad....I just realized what you link was that you provided. I'll find the logs.

**Comment by sgonzales3478** [comment]
I believe I got the right ones.

[2024-06-10-2.log.gz](https://github.com/user-attachments/files/15765183/2024-06-10-2.log.gz)
[debug-1.log.gz](https://github.com/user-attachments/files/15765185/debug-1.log.gz)

**Comment by sgonzales3478** [comment]
I didn't see one called "latest" in the log folder, but when I did a command search on my Mac I did find these as well.

[latest.log](https://github.com/user-attachments/files/15765204/latest.log)
[debug.log](https://github.com/user-attachments/files/15765215/debug.log)

**Comment by sgonzales3478** [comment]
Hey there, so sorry for any confusion I may have caused. I wanted to make sure that I definitely got you the right ones, so I went back in started the process all over with another new world. This time I did get the two files to show up in the log folder.

[latest.log](https://github.com/user-attachments/files/15766044/latest.log)
[debug.log](https://github.com/user-attachments/files/15766047/debug.log)

**Comment by sgonzales3478** [comment]
I completely forgot to do the upload instructions from your link. So sorry, this is all new to me as I've never done this before, so I'm learning as we go.

https://paste.feed-the-beast.com/view/df95063a

**Comment by Gaz492** [comment]
Its no problem at all, the only issue is I can only see the text ```latest.log
debug.log``` in the link you provided, you can also just upload the 2 files here by dragging and dropping them onto the text box.

**Comment by sgonzales3478** [comment]
Ahh....I thought I wasn't supposed to do that, so I had removed them when I saw what your link instructions mentioned. Here they are.
[l
[debug.log](https://github.com/user-attachments/files/15768415/debug.log)
atest.log](https://github.com/user-attachments/files/15768398/latest.log)

**Comment by Gaz492** [comment]
Thanks,

It looks like JAMD ran into an error while parsing its config, so I can try reproducing this, may I ask what mods you disabled as it may be related to that.

**Comment by sgonzales3478** [comment]
Oh boy.....quite a handful of them! Is there any way that I can share something with you that shows which are disabled? Otherwise, I'll take some time after a bit here and write down all the ones that are disabled.

**Comment by Gaz492** [comment]
If you click on the pack in the FTB App, then go to the mods tab, it may show the ones you disabled there.
I have spoken to the author of JAMD, and they suspect this is indeed caused by the mods that have been disabled.

You could try deleting the mining.json config found in `config` > `jamd` this will then generate a fresh config for the mod, but it will likely only contain vanilla ores.

**Comment by sgonzales3478** [comment]
So going through the list in the app these are all the ones I have disabled. Quite a big list!

1. Ad Astra
2. Ad Astra: Giselle Add-on
3. Advanced Generators
4. Advanced Peripherals
5. Aether Add-on: Enhanced Extinguishing
6. Angel Block Renewed
7. Antsportation
8. Apotheosis
9. Apothic Attributes
10. Applied Botanics Add-on
11. Applied Cooking
12. Ars Creo
13. Better Than Bunnies
14. Better Than Llamas
15. Botania
16. CC: Tweaked
17. Cooking for Blockheads
18. Create
19. Create: Teleporters
20. Create Chunkloading
21. Create Crafts & Additions
22. Create Slice & Dice
23. CreatTweaker
24. Deep Resonance
25. Farming for Blockheads
26. Forbidden and Arcanus: Path of Knowledge
27. Gateways to Eternity
28. KubeJS Create
29. Modular Force Field Systems
30. Modular Golems
31. More Red x CC: Tweaked Compat
32. Peripheralium
33. Productive Bees
34. Railcraft Reborn
35. Rats
36. Rechiseled: Create
37. Refined Cooking
38. The Aether
39. The Twighlight Forest
40. Tome of Blood: Rebirth
41. Topicraft
42. UnlimitedPeripheralWorks
43. Wormhole (Portals)

**Comment by sgonzales3478** [comment]
FYI....I did try deleting the mining.json config that you mentioned yesterday, went back into my test world and it actually did end up working and generated ores. And it also looks like it generated more than just the vanilla ores because I did see ores from Mekanism, Thermal Series, RFToolsBase, and even XyCraft World.

I mad a backup of my main world save and tried it in there as well and the mining dimension also generated ores finally in that world too.
